### ISSUE TITLE
🐛 [bug] - Create alerts exercise seems to be incomplete

### ISSUE THREAD

**Issue by alexpdp7** [rht-labs/tech-exercise#87]
### 📝 Description

Step 6 causes an alert, but there are no steps to see the alert and/or verify it's fine. I couldn't see the alert everywhere, FWIW.

### 🚶 Steps to reproduce

Execute up to step 6.

### 🧙‍♀️ Suggested solution

Instruct the user to go to the proper place to view the alert. Maybe, remove the `dd`ed file.

Also note that there are no instructions to exit from `rsh` here:

https://github.com/rht-labs/tech-exercise/blame/main/docs/4-return-of-the-monitoring/2-create-alerts.md#L80

Which may be confusing.

**Comment by ckavili** [comment]
yeah we should add that the alert firing is visible on OpenShift UI main page..on it :)
